webpack.config.js

```diff
@@ -254,25 +253,14 @@ if ( process.env.DASHBOARD ) {
 	webpackConfig.plugins.unshift( new DashboardPlugin() );
 }
 
-if ( process.env.WEBPACK_OUTPUT_JSON ) {
+if ( process.env.WEBPACK_OUTPUT_JSON || ! isDevelopment ) {
```

Comment:
I think the `WEBPACK_OUTPUT_JSON` flag is not needed any more. Its only purpose was to force using the uglify webpack plugin in a production build, so that the `analyze-bundles` output takes minification into account. Now when uglify is the default, we can remove it.

Reply:
makes sense to me. removed in https://github.com/Automattic/wp-calypso/pull/19918/commits/26e2b1dbedb1eb9239ab9ef01ac7ec8992cdb76a